examples/shopping-cart/test/reducers/index.spec.js

```diff
@@ -0,0 +1,85 @@
+import expect from 'expect'
+import { getTotal, getCartProducts } from '../../reducers'
+
+describe('reducers', () => {
```

Comment:
Maybe worth changing to `selectors`? They are grouped with reducers in the code for convenience but we should probably label them differently.